blockly/build.gradle

```
@@ -0,0 +1,14 @@
+plugins {
+    id 'java'
```

```suggestion
    id 'application'
```
haben wir bisher auch so